src/course-outline/hooks.jsx

```diff
@@ -81,23 +97,37 @@ const useCourseOutline = ({ courseId }) => {
     }
   }, [reIndexLoadingStatus]);
 
+  const {
+    visibility: learnMoreVisibilityUrl,
```

Comment:
maybe would be better to get these url separately in `OutlineSideBar`, `OutlineSideBar`, `StatusBar` instead of passing from the props

Reply:
Done